src/main/java/jpass/data/DocumentHelper.java

```diff
@@ -138,7 +138,7 @@ public void writeDocument(final Entries document) throws DocumentProcessExceptio
             if (this.key == null) {
                 outputStream = new FileOutputStream(this.fileName);
             } else {
-                outputStream = new GZIPOutputStream(new CryptOutputStream(new FileOutputStream(this.fileName), this.key));
+                outputStream = new CryptOutputStream(new FileOutputStream(this.fileName), this.key);
```

Comment:
Similarly here, instead of removing the GZIPOutputStream you could wrap the FileOutputStream e.g. into a SaltOutputStream.